org.gnome.Evince.json

```diff
@@ -153,9 +183,43 @@
             ],
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://download.gnome.org/sources/gnome-desktop/3.38/gnome-desktop-3.38.0.tar.xz",
-                    "sha256": "089dbbe3c66fe5575659a4a385d5d4bbd99cf637034df317f21cf586b5dd6b90"
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gnome-desktop.git",
+                    "tag": "3.38.4",
+                    "commit": "8662b94232a2d65f1aa76acb3d9e20a8b70489cf",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 13127,
+                        "tag-template": "$version"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "libhandy",
```

Comment:
Adding libhandy should be a separate commit too.

Reply:
It's a dependency of newer versions of Evince... How can I update Evince without adding it, or adding it without bump the version which is useless for the old version?

Reply:
The next release of Evince will use the next version of the SDK which includes libhandy already.

Reply:
Indeed. No need to add `libhandy` here.

Reply:
> The next release of Evince will use the next version of the SDK which includes libhandy already.

That's a good news though. Not sure how long will it take.